aws/data_source_aws_organizations_delegated_services_test.go

```diff
@@ -0,0 +1,44 @@
+package aws
+
+import (
+	"testing"
+
+	"github.com/aws/aws-sdk-go/service/organizations"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func TestAccDataSourceAwsOrganizationsDelegatedServices_basic(t *testing.T) {
+	var providers []*schema.Provider
+	dataSourceName := "data.aws_organizations_delegated_services.test"
+	dataSourceIdentity := "data.aws_caller_identity.delegated"
+
+	resource.ParallelTest(t, resource.TestCase{
```

Comment:
Since this test is for settings at the account level, it should be run in parallel with other tests

```suggestion
	resource.Test(t, resource.TestCase{
```